crud/select/executor.lua

```diff
@@ -45,16 +45,22 @@ if has_keydef then
     generate_value = function(after_tuple, scan_value, index_parts, tarantool_iter)
         local cmp_operator = select_comparators.get_cmp_operator(tarantool_iter)
         local key_def = keydef_lib.new(index_parts)
+        if #scan_value == 0 and after_tuple ~= nil then
```

Comment:
I have no idea how to test the internal behavior. I'm not sure if the internal behavior needs to be tested too.
But we already have integration tests for select(nil, {after}).

Reply:
Because when I first saw this commit, my first thought was "before this patch `select(nil)` with `after` extracted all records, now it extracts only records starting after `after`"